Makefile

```diff
@@ -0,0 +1,22 @@
+lint: lint-go  lint-templates lint-static
+
+format: format-go format-static format-templates
+
+format-go:
+	go fmt ./...
+
+lint-go:
+	golangci-lint run
+
+lint-static:
+	npx prettier --check "static/**/*.{js,css}"
+	npx eslint static/javascript/*.js
+
+format-static:
+	npx prettier "static/**/*.{js,css}" --write
+
+lint-templates:
```

Comment:
As we discussed, djlint requires python install so some changes are required.

Reply:
Updated the relevant files to include this change 👍

Reply:
As discussed, will move from Pipenv to Poetry

Reply:
Updated 👍